core/src/main/java/org/mobilitydata/gtfsvalidator/notice/NoticeSchemaGenerator.java

```diff
@@ -123,8 +114,9 @@ static Map<String, Map<String, Class<?>>> contextFieldsInPackages(List<String> p
       throws IOException {
     // Return a sorted TreeMap for stable results.
     Map<String, Map<String, Class<?>>> contextFieldsByNotice = new TreeMap<>();
-    for (Class<Notice> noticeClass : findNoticeSubclasses(packages)) {
-      contextFieldsByNotice.put(noticeClass.getSimpleName(), contextFieldsForNotice(noticeClass));
+
+    for (Class<Notice> noticeClass : ClassGraphDiscovery.discoverNoticeSubclasses(packages)) {
+      contextFieldsByNotice.put(Notice.getCode(noticeClass), contextFieldsForNotice(noticeClass));
```

Comment:
AFAIU the returned Map has notice code as its instead of class name, could you please update the method comment?

Reply:
Done